bamboo/unit_tests/test_checkpoint.py

```diff
@@ -0,0 +1,38 @@
+import os, pytest
+
+def test_checkpoint_lenet(exe, dirname):
+    return_code_nockpt = os.system('salloc -N1 srun -n1 %s --model=%s/model_zoo/tests/model_lenet_mnist_ckpt.prototext --optimizer=%s/model_zoo/optimizers/opt_sgd.prototext --reader=%s/model_zoo/data_readers/data_reader_mnist.prototext --num_epochs=2' % (exe, dirname, dirname, dirname))
```

Comment:
I don't think `nockpt` is a great abbreviation. I see that it means "no checkpoint" but that is not immediately clear. "ckpt" isn't a great abbreviation of "checkpoint" either. It might be better to just not abbreviate.